react/AddressForm.tsx

```diff
@@ -219,7 +218,7 @@ const AddressForm: React.FC<AddressFormProps> = ({
                   }))
                 }
 
-                const value = address[fragment.name] as string
+                const value = address[fragment.name] ?? ''
```

Comment:
What happens if the `fragment.name` is `geoCoordinates` (array of float) or `isDisposable` (boolean)?

Reply:
these fields shouldn't be editable, so we need to "curate" the countries-data repo to always mark them as `hidden: true`

Reply:
also, even before this PR you can see that the `value` variable was being "converted" to string with `as string`, so this shouldn' t change the behavior of the code compared to what is was before